packages/components/src/components/domainproperties/NameAndLinkingOptions.tsx

```diff
@@ -46,7 +46,7 @@ export class NameAndLinkingOptions extends React.PureComponent<NameAndLinkingPro
             <>
                 Use this to change the display of the field value within a data grid into a link. Multiple formats are supported, which allows ways to easily substitute and link to other locations in LabKey.
                 <br/><br/>
-                Learn more about using <a target="_blank" href={LK_URL_ENCODING_DOC}>URL Formatting Options</a>
+                Learn more about using <a target="_blank" href={LK_URL_ENCODING_DOC}>URL Formatting Options</a>.
```

Comment:
Similarly, this link and, I think all those that use LK_BASE_HELP_URL, should be constructed from LABKEY.helpLinkPrefix (but these changes are probably for a different PR).

Reply:
done